packages/datadog-plugin-kafkajs/src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+function createWrapProducer (tracer, config) {
+  return function wrapProducer (createProducer) {
+    return function producerWithTrace () {
+      const serviceName = config.service || `${tracer._service}-kafka`
+      const createdProducer = createProducer.apply(this, arguments)
+
+      const send = createdProducer.send
+
+      const tags = {
+        'service.name': serviceName,
+        'span.kind': 'producer',
+        'span.type': 'queue',
+        component: 'kafka'
+      }
+
+      createdProducer.send = tracer.wrap('kafka.producer.send', { tags }, function (...args) {
+        const { topic, messages } = args[0]
+        const currentSpan = tracer.scope().active()
+
+        currentSpan.addTags({
+          'resource.name': `produce to ${topic}`,
```

Comment:
This should probably be just the topic since the operation name already implies that it's producing.